data/json/recipes/food/brewing.json

```diff
@@ -1,4 +1,19 @@
 [
+  {
+    "type": "recipe",
+    "result": "brew_mycus_wine",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_BREWING",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "time": 40000,
+    "autolearn": true,
+    "book_learn": [ [ "brewing_cookbook", 1 ], [ "family_cookbook", 2 ] ],
```

Comment:
Probably shouldn't be learned from books, because mycus fruits are only post-cataclysm. At least not from pre cataclysm books

Reply:
Thanks for both, I will fix this

Reply:
I have changed both, as requested.